src/classes/ConfData.js

```diff
@@ -111,8 +111,8 @@ class ConfData {
 			_initProperty('enable_click2play', true);
 			_initProperty('enable_click2play_social', true);
 			_initProperty('enable_human_web', !IS_CLIQZ && !IS_FIREFOX);
-			_initProperty('enable_metrics', false);
-			_initProperty('enable_offers', !IS_CLIQZ && !IS_FIREFOX && !IS_ANDROID);
+			_initProperty('enable_metrics', BROWSER_INFO.name === 'ghostery_desktop');
```

Comment:
Again, could this be a race?

Reply:
Same as above, it does evaluate in order but the possibility for a race condition technically exists.